Восстановил, после сломного репозитория

Oleg667 opened the pull request:
Созданы API тесты.
*восстановил после сломанного репозитория

Oleg667 committed:
Восстановил, после сломного репозитория

pomegranate282922 commented on src/test/java/OrderTest.java:
```diff
@@ -0,0 +1,208 @@
+    @Test
+    @DisplayName("#2 - Создание заказа с авторизации")
+    @Description("Только ингредиенты ")
+    public void test_CreateOrder_Authorization() {
```
Методы именуем camelCase 
Нужно исправить

Oleg667 replied:
Исправил

pomegranate282922 commented on src/test/java/OrderTest.java:
```diff
@@ -0,0 +1,208 @@
+
+        } finally {
+            // Удаляем созданного пользователя
+            cleanUp(rawAccessToken);
```
Вместо finally намного лаконичнее использовать after each 
Можно улучшить

Oleg667 replied:
Переделал

pomegranate282922 commented on src/test/java/UserTest.java:
```diff
@@ -0,0 +1,205 @@
+    @DisplayName("#3 - Авторизация пользователя без пароля → 401")
+    @Description("Проверка, что API возвращает статус 401, Ответ: \"email or password are incorrect\" ")
+    public void test3_loggedUserWithoutPassword_ShouldReturn401() {
+        String json = "{"
```
Не передаем тело запроса через строку. Используем объект или мапу 
Нужно исправить

Oleg667 replied:
Исправил

pomegranate282922 commented on src/test/java/config/Config.java:
```diff
@@ -0,0 +1,36 @@
+    public static final String USER_DEL_API = "/api/auth/user"; //DEL удаление пользователя
+    public static final String USER_PATCH_API = "/api/auth/user"; //DEL удаление пользователя
+    public static final String INGRADIENT_API = "/api/ingredients"; //GET получение списка инградиентов
+
```
Отлично что вынесены константы

pomegranate282922 commented on src/test/java/steps/OrderSteps.java:
```diff
@@ -0,0 +1,102 @@
+                .response();
+    }
+    @Step("Список заказов пользователя")
+    public static Response OrderUser(String accessToken) {
```
camelCase 
Нужно исправить

Oleg667 replied:
Исправил

pomegranate282922 reviewed (commented):
Работа хорошая! Все тесты реализованы. Лаконичный код. 
замечания:
1. Нейминг
2. Тело запроса передаем через объект или мапу

Oleg667 committed:
Исправл замечания:
- два имени согласно camelCase
- тело запроса теперь через мапу
- заменил везде finally на after each

pomegranate282922 reviewed (approved):
Отлично, зачет!
